src/features/navigation/sidebar/VerticalMenu.tsx

```diff
@@ -393,7 +393,9 @@ export const VerticalMenu: React.FC<any & { history: History }> = ({
                   }
                 >
                   <House />
-                  <MenuText>Dashborad {isHintingOption && '(D)'}</MenuText>
+                  <HotkeyView hotkey="D" hint={isHintingOption}>
```

Comment:
Maybe even more clear?
```suggestion
                  <HotkeyView hotkey="D" hinting={isHintingOption}>
```